web/impact/impact/urls.py

```diff
@@ -123,6 +123,9 @@
     url(r'^people/(.*)/$', TemplateView.as_view(
         template_name='front-end.html'),
         name="entreprenuer_profile"),
+    url(r'^startups/$', TemplateView.as_view(
+        template_name='front-end.html'),
+        name="startup_profile"),
```

Comment:
This should be `startup_directory` looks like the people directory was also wrongly named `entreprenuer_profile`

Reply:
sorted.